### DISCUSSION TITLE
When new ports are shown on https://vcpkg.io/en/packages.html ?

### DISCUSSION THREAD

**Discussion by ilya-lavrenov** [microsoft/vcpkg#32075] [General]
I've submitted a PR with new port and it's merged now (thanks for great review and help).

I've tried to open https://vcpkg.io/en/packages.html and type name of my port and have not found it.
It's turned out that even new ports from [last release](https://github.com/microsoft/vcpkg/releases/tag/2023.04.15) not shown on the site, e.g. type `arsenalgear` and you will not see any results.

Even new versions of existing ports are also not shown.

What is the process of publishing updated information in the packages page?

Thanks,
Ilya.

**Comment by Osyotr** [comment]
I believe the site updates are triggered manually by some vcpkg team members.
Personally I would advice using https://vcpkg.link as the most convenient and up-to-date vcpkg database.

**Reply by ilya-lavrenov** [reply]
Thanks, but I don't see new ports even on https://vcpkg.link/
Does it also have some delay?

**Reply by autoantwort** [reply]
<img width="525" alt="image" src="https://github.com/microsoft/vcpkg/assets/41973254/3910a8df-318f-4be9-986f-ae75b8b0e0b2">
I guess it runs once a day

**Comment by autoantwort** [comment]
Someone has to manually update it [here](https://github.com/vcpkg/vcpkg.github.io)
